### ISSUE TITLE
Group 4: Early Women Graduates using RawGraphs.io

### ISSUE THREAD

**Issue by naomiwells85** [SASDigitalHumanitiesTraining/Visualisation#28]

**Comment by aghague** [comment]
I don't know enough about the history of higher education for women to fully understand the dataset, nor do I understand exactly how the eigencentrality value was calculated and what it actually does, so I am not entirely sure which of RAWGraphs visualisation options are, in fact, useful. For example, in my attempt to represent the volume of degrees by subject I ended up with "pass science", "BSc/Sci/Maths" and "Physics" as three different categories rather than see them lumped together in a general category called "science".

**Comment by Pylaemenes** [comment]
A beeswarm based on subject and eigencentrality (I understand his as "influencers" of an older sort. 
<img width="1240" alt="Screen Shot 2020-12-10 at 6 29 32 AM" src="https://user-images.githubusercontent.com/75645297/101766862-669b2c80-3ab1-11eb-928a-23226b24344c.png">

I don't know what exactly this could show in this case, but I can see the usefulness if one understands the data itself better.

**Comment by Pylaemenes** [comment]
A second "BoxPlot" I think gives a better sense of eigencentrality based on school and degree:

<img width="930" alt="Screen Shot 2020-12-10 at 6 46 58 AM" src="https://user-images.githubusercontent.com/75645297/101768528-aa8f3100-3ab3-11eb-91a4-e5067c8d0a33.png">

**Comment by Pylaemenes** [comment]
Can't figure out why the years are not in order!

<img width="949" alt="Screen Shot 2020-12-10 at 6 51 48 AM" src="https://user-images.githubusercontent.com/75645297/101768922-3b660c80-3ab4-11eb-976d-0e655e3e1c95.png">

**Comment by aghague** [comment]
@Pylaemenes 

> Can't figure out why the years are not in order!
> 
> <img alt="Screen Shot 2020-12-10 at 6 51 48 AM" width="949" src="https://user-images.githubusercontent.com/75645297/101768922-3b660c80-3ab4-11eb-976d-0e655e3e1c95.png">

Yes, same here. Also: the data I have has large gaps in it (i.e., there are very few complete entries) - so I'm not sure how useful *any* of my attempted visualisations are.

**Comment by aghague** [comment]
@Pylaemenes 

> A second "BoxPlot" I think gives a better sense of eigencentrality based on school and degree:
> 
> <img alt="Screen Shot 2020-12-10 at 6 46 58 AM" width="930" src="https://user-images.githubusercontent.com/75645297/101768528-aa8f3100-3ab3-11eb-91a4-e5067c8d0a33.png">

Yes, you're right, this does give a better sense of the overall picture, and I would say it's a lot better than what I've been trying to do. I do wish it was possible to label things more clearly, though. There's not much you can do about it short of editing the dataset, I think, given that not even the tutorials manage to use/display labels well.

**Comment by Pylaemenes** [comment]
Because I'm a classicist, I also went into Tableau and did some Roman amphitheaters by capacity

<img width="920" alt="Screen Shot 2020-12-10 at 7 49 37 AM" src="https://user-images.githubusercontent.com/75645297/101774709-78360180-3abc-11eb-89df-f173206b6e27.png">

**Comment by aghague** [comment]
I tried to use RG to track which schools the students came from in each cohort and what they went on to study, but the result is too messy for my liking, and the connections are only pair-based, not multi-step; highly disappointing. Plus, the years are still not in chronological order - which annoys me somewhat. 

![RG graduates arrival degrees](https://user-images.githubusercontent.com/74906423/101776178-741fd900-3ae8-11eb-800e-0b6d48031ead.png)
